### PR TITLE
reconcile users

### PR DESCRIPTION
**Pull request by grudelsud**
scripts to async manage users on slack + website + groups

### CODE REVIEW

File: scripts/admin/users/utils.py [hunk 1]
```diff
@@ -24,57 +16,202 @@
...
+
+SLACK_FILE = 'slack-devconteam-members.csv'
+SLACK_FIELDS = ['username', 'email', 'status', 'billing-active', 'has-2fa', 'has-sso', 'userid', 'fullname', 'displayname']
+
+GOOGLE_GROUP_FILE = 'devconlondon.csv'
+
+MEMBERS_EXPORT_FILE = 'devcon.members.csv'
+MEMBERS_EXPORT_FIELDS = ['uid', 'name', 'email', 'referrer', 'date', 'linkedin', 'github', 'twitter', 'role']
+
+email_body = """<strong>A friendly poke from Devcon.network</strong>
```
Comment (grudelsud) [inline scripts/admin/users/utils.py]: can automatically send this email to all users who ARE on slack but NOT on website.

Comment (silviopaganini) [reply scripts/admin/users/utils.py]: nice

**Comment by silviopaganini** [comment]
merge?

**Comment by grudelsud** [comment]
not yet, I'm trying to have a look at automatic google group invite, which doesn't seem to work for normal groups (only business on gsuite). will update the script to ease the process of inviting 10 users at a time. will merge as soon as done

**Pull request merged**